src/Plugin/Field/FieldWidget/EnhancedButtonLinkWidget.php

```diff
@@ -69,6 +71,7 @@ public function formElement(FieldItemListInterface $items, $delta, array $elemen
       '#default_value' => isset($items[$delta]->options['status']) ? $items[$delta]->options['status'] : '',
       '#description' => $this->t('Select the status of the button.'),
       '#options' => [
+        'default' => $this->t('default'),
```

Comment:
Please update 'default' to 'Default', so it should be same as:
`$this->t('Default'),`
and do the same for title of 'enabled' and 'disabled'